client/src/utils.ts

```diff
@@ -0,0 +1,27 @@
+import * as vscode from 'vscode';
+
+export const isWindows = process.platform === 'win32';
+
+function isPowershell() {
+  try {
+    const config: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration(
+      'ElmLS',
```

Comment:
It seems like we have been using `elmLS` on the server side for settings

Reply:
Looking into right now, I don't think it's needed at all. In this particular case we don't want part of the config related to Elm but rather whole config object.